practice-one/src/index.html

```diff
@@ -6,9 +6,11 @@
     <title>Document</title>
     <link rel="stylesheet" href="assets/css/global.css" type="text/css"/>
     <link rel="stylesheet" href="assets/css/header.css" type="text/css"/>
+    <link rel="stylesheet" href="assets/css/footer.css" type="text/css"/>
     <link rel="stylesheet" href="assets/css/reset.css" type="text/css"/>
```

Comment:
I think if I place the reset file here, the reset file will override any styles from the previously loaded CSS files. This is because CSS follows the "Cascading" principle, where styles declared later in the stylesheet override styles declared earlier. So my solution is that we should the reset file at the beginning, before other stylesheets, to establish a consistent baseline for my styles. 

I fixed it
![image](https://github.com/HoangLiarPhuongLe/html_css_training/assets/110210274/c4b465e5-56aa-4c41-9681-dcf7e6bee023)